src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final Pair<K, V>[] pairs;
+    private int capacity = 0;
+
+    public StorageImpl() {
+        this.pairs = new Pair[10];
```

Comment:
```suggestion
        pairs = new Pair[10];
```
and what should we do with magic numbers?